packages/ttf_hanazono.rb

```diff
@@ -0,0 +1,29 @@
+require 'package'
+
+class Ttf_hanazono < Package
+  description 'A free Japanese kanji font, which contains about 78,685 characters (and 2 SPACEs) defined in ISO/IEC 10646 standard / the Unicode standard.'
+  homepage 'http://fonts.jp/hanazono' # From ArchLinux PKGBUILD, could not actually resolve site
+  @_ver = 20170904
+  @_revision = 68253
+  version "#{@_ver}-r#{@_revision}"
```

Comment:
`@_revision` not needed?

Reply:
what if they change the revision but not the version

Reply:
I'm thinking that is not a revision.

Reply:
don't ask me I didn't invent the versioning i just follow it :shrug: